downstream/titles/security-guide/docinfo.xml

```diff
@@ -1,10 +1,10 @@
 <title>Red Hat Ansible Security Automation Guide</title>
 <productname>Red Hat Ansible Automation Platform</productname>
 <productnumber>2.0</productnumber>
```

Comment:
Is this title meant for 2.0?

Reply:
Thanks for the review, @TarikFD. This fix is to be backported into 2.1.
Good catch!